lib/tasks/elasticsearch.rake

```diff
@@ -14,15 +14,15 @@ namespace :elasticsearch do
     res = Rubygem.__elasticsearch__.client.count index: idx
     puts "Count before import: #{res['count']}"
 
-    new_idx = "#{idx}-#{Time.zone.today.strftime('%Y%m%d')}"
+    new_idx = "#{Rubygem.index_name}-#{Time.zone.today.strftime('%Y%m%d%H%M')}"
     Rubygem.import index: new_idx, force: true
     res = Rubygem.__elasticsearch__.client.count index: new_idx
     puts "Count after import: #{res['count']}"
 
     Rubygem.__elasticsearch__.delete_index! index: idx
     Rubygem.__elasticsearch__.client.indices.update_aliases body: {
       actions: [
-        { add: { index: new_idx, alias: idx } }
+        { add: { index: new_idx, alias: Rubygem.index_name } }
```

Comment:
You need to remove `Rubygem.__elasticsearch__.delete_index! index: idx` and previous alias.